frame/dvm/src/lib.rs

```diff
@@ -135,6 +134,7 @@ decl_storage! {
 			<Module<T>>::store_block(false);
 
 			// Initialize the storage schema at the well known key.
+			#[cfg(not(feature = "try-runtime"))]
```

Comment:
Why here

Reply:
Since `PALLET_ETHEREUM_SCHEMA` is None in an existing network.  The test goal is to ensure this value from None to V1 after migration.